vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/ApiSteps.java

```diff
@@ -241,15 +241,15 @@ private static Object castType(String typeName, String value)
         }
     }
 
-    private HttpRequestBuilder prepareHttpRequestBuilder(HttpMethod httpMethod, String endpoint)
+    public HttpRequestBuilder prepareHttpRequestBuilder(HttpMethod httpMethod, String endpoint)
     {
         HttpRequestBuilder httpRequestBuilder = HttpRequestBuilder.create().withHttpMethod(httpMethod)
                 .withEndpoint(endpoint).withHeaders(httpTestContext.pullRequestHeaders());
         httpTestContext.pullRequestEntity().ifPresent(httpRequestBuilder::withContent);
         return httpRequestBuilder;
     }
 
-    private HttpResponse executeHttpCallSafely(HttpRequestBuilder builder) throws IOException
+    public HttpResponse executeHttpCallSafely(HttpRequestBuilder builder) throws IOException
```

Comment:
multiple public methods in classes implementing steps bring a lot of confusion and ambiguity, please, consider refactoring avoid this approach